lib/api/_loaders/ensure.js

```diff
@@ -260,8 +260,10 @@ class EnsureAssertionLoader extends BaseLoader {
         namespace = parent.ensure = parent.ensure || {};
       }
 
+      const abortOnFailure = this.nightwatchInstance.settings.globals.abortOnAssertionFailure;
```

Comment:
```suggestion
      const { globals: { abortOnAssertionFailure } } = this.nightwatchInstance.settings;
```
Now, we can use `abortOnAssertionFailure` directly